x-aeson/ambiata-x-aeson.cabal

```diff
@@ -15,7 +15,11 @@ library
                        aeson                           >= 0.8        && < 0.12
                      , base                            >= 3          && < 6
                      , bytestring                      == 0.10.*
+                     , containers                      == 0.5.*
                      , text                            >= 1.1        && < 1.3
+                     , time                            >= 1.5        && < 2
```

Comment:
Do we still need 1.4? I'd like to use the re-exported `defaultTimeLocale` instead of CPP